drivers/sensor/pmw3360/pmw3360.c

```diff
@@ -842,37 +797,33 @@ static int pmw3360_init_irq(const struct device *dev)
 	return err;
 }
 
-static int pmw3360_init_spi(struct pmw3360_data *dev_data)
-{
-	dev_data->spi_dev = device_get_binding(PMW3360_SPI_DEV_NAME);
-	if (!dev_data->spi_dev) {
-		LOG_ERR("Cannot get SPI device");
-		return -ENXIO;
-	}
-
-	return 0;
-}
-
 static int pmw3360_init(const struct device *dev)
 {
 	struct pmw3360_data *dev_data = &pmw3360_data;
+	const struct pmw3360_config *dev_config = dev->config;
 	int err;
 
 	ARG_UNUSED(dev);
 
 	k_work_init(&dev_data->trigger_handler_work, trigger_handler);
 
-	err = pmw3360_init_cs(dev_data);
-	if (err) {
-		return err;
+	if (!spi_is_ready(&dev_config->bus)) {
+		LOG_ERR("SPI device not ready");
+		return -ENODEV;
 	}
 
-	err = pmw3360_init_irq(dev);
+	if (!device_is_ready(config->cs_gpio.port)) {
+		LOG_ERR("SPI CS device not ready");
+		return -ENODEV;
+	}
+
+	err = gpio_pin_configure_dt(&config->cs_gpio, GPIO_OUTPUT);
```

Comment:
`GPIO_OUTPUT_INACTIVE` ? To make sure pin is in proper state

Reply:
adjusted